jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/BPMN2DataServicesReferencesTest.java

```diff
@@ -510,7 +521,41 @@ public void testSignalsAndGlobals() throws IOException {
                     "person".equals(globalName) || "name".equals(globalName) );
         }
 
+        assertNotNull("Signals description should not be null", procDef.getSignalsDesc());
+        assertNotNull("Messages description should not be null", procDef.getMessagesDesc());
+        assertTrue("Signal description should be empty", procDef.getSignalsDesc().isEmpty());
+        assertTrue("Messages description should be empty", procDef.getMessagesDesc().isEmpty());
+
         // cleanup
         processService.abortProcessInstance(procInstId);
     }
+
+    @Test
+    public void testSignalsDesc() throws IOException {
```

Comment:
Checked exception is not thrown within this method
```suggestion
    public void testSignalsDesc() {
```